CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ Next
 * Added TODO list and enhanced README documentation - [@ptitvert](https://github.com/ptitvert)
 * Your contribution here.
 
+1.2 (2/1/2014)
+================
+* Added support for CPU load in OSX
```

Comment:
This should be under Next and should just be a bullet with your name (same format as above please).